src/qml/qgismobileapp.qml

```diff
@@ -809,8 +808,8 @@ ApplicationWindow {
   Controls.BusyIndicator {
     id: busyIndicator
     anchors { right: alertIcon.left; top: parent.top }
-    width: 36 * dp
-    height: 36 * dp
+    width: 96 * dp
```

Comment:
Is this related to this change?
I think it's unfortunately not that simple, it looks good on low-res devices and bad on high-res devices, so I think we need to do something else than just increase the factor. Can we just keep it out for now?

Reply:
Yes. Actually it has been more for size testing. We can remove it.